### PR TITLE
remove current_month field and rotating_message_table

### PR DESCRIPTION
**Pull request by taddes**
Closes [SYNC-3703](https://mozilla-hub.atlassian.net/browse/SYNC-3703)

[SYNC-3703]: https://mozilla-hub.atlassian.net/browse/SYNC-3703?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/mod.rs [hunk 1]
```diff
@@ -284,41 +284,7 @@ impl WebPushClient {
...
-                .incr_with_tags("ua.expiration")
-                .with_tag("code", "105")
-                .with_tag("reason", "invalid_current_month")
-                .send();
-            app_state.db.remove_user(&user.uaid).await?;
-            return Ok(None);
-        }
-    }
-
+pub async fn process_existing_user(user: &User) -> DbResult<Option<ClientFlags>> {
```
Comment (jrconlin) [inline autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/mod.rs:287]: Heh, might be adding `app_state` back later when some of the other PRs land, but it's fine pulling it now.

File: autopush-common/src/db/dual/mod.rs [hunk 2]
```diff
@@ -0,0 +1,597 @@
+//! Dual data store.
```
Comment (jrconlin) [inline autopush-common/src/db/dual/mod.rs:1]: Welp, this will need to be refactored 😆 
https://github.com/mozilla-services/autopush-rs/pull/715

**Review by jrconlin — CHANGES_REQUESTED** [review]
Just need to rectify this PR with the removal of `dual` mode.

File: autopush-common/src/db/dual/mod.rs (hunk 2 above)
Comment (taddes) [reply autopush-common/src/db/dual/mod.rs:1]: Hmmm I wonder if this found it's way back in when rebasing. Ack, I will take it out.

Comment (taddes) [reply autopush-common/src/db/dual/mod.rs:1]: And yet, in https://github.com/mozilla-services/autopush-rs/pull/715 it's missing...

**Review by pjenvey — APPROVED** [review]

**Review by jrconlin — APPROVED** [review]

**Pull request merged**